src/screens/SettingsSheet/components/NetworkSection.tsx

```diff
@@ -18,22 +18,18 @@ interface NetworkSectionProps {
 
 const NetworkSection = ({ inDevSection }: NetworkSectionProps) => {
   const { chainId, testnetsEnabled } = useAccountSettings();
-  const resetAccountState = useResetAccountState();
   const loadAccountData = useLoadAccountData();
-  const initializeAccountData = useInitializeAccountData();
   const dispatch = useDispatch();
 
   const onNetworkChange = useCallback(
     async (chainId: ChainId) => {
-      await resetAccountState();
```

Comment:
`resetAccountState` only has the resetting of WC v1 requests now - given that this doesn't change based on accountSetting's `network` anymore since we don't support testnets at the moment, and given that we will be removing WC v1, I am preemptively removing this here